Source/santad/Santad.mm

```diff
@@ -151,11 +151,13 @@ void SantadMain(std::shared_ptr<EndpointSecurityAPI> esapi, std::shared_ptr<Logg
 
     access_authorizer_client.fileAccessBlockCallback = ^(
         SNTFileAccessEvent *event, NSString *customMsg, NSString *customURL, NSString *customText) {
+      SNTConfigState *cs = [[SNTConfigState alloc] initWithConfig:[SNTConfigurator configurator]];
```

Comment:
Note for future self: This is fine for now, but when we actually make config states valid for the life of processing an event, the config state will need to come as an argument to this block. Nothing to do about this today though.

Reply:
I added a TODO